release-notes/8.0/supported-os.md

```diff
@@ -80,7 +80,7 @@ Other distributions are supported at best effort, per [.NET Support and Compatib
 
 OS                            | Version                   | Architectures     |
 ------------------------------|---------------------------|-------------------|
-[macOS][macOS]                | 10.15+                    | x64, Arm64        |
+[macOS][macOS]                | 12.0                    | x64, Arm64        |
```

Comment:
Nit: This should be `12+`. We have plus on all other versions. Apple dropped minor versions in the versioning scheme of their OSes.